calyx-backend/Cargo.toml

```diff
@@ -18,6 +18,7 @@ string-interner.workspace = true
 itertools.workspace = true
 linked-hash-map.workspace = true
 serde = { workspace = true }
+serde_json.workspace = true
```

Comment:
I am ignorant enough of Rust workspace setups to not exactly know the difference between using `serde_json.workspace = true` here vs. just doing this right here:

```toml
serde_json = "1.0"
```

…and omitting this from the workspace-global `Cargo.toml`. Presumably there is a difference but I'm not sure what it is! Maybe this is the right thing; just thought I'd double-check this was the intended strategy (vs. what we do for `csv` and `vast` below).

Reply:
Maybe @rachitnigam has an opinion on this? Tbh I've just been pattern matching from stuff I've seen around, the details of these workspaces elude me.

Reply:
It probably doesn't matter much! I guess the upside of what you have now is that anyone else in the tree who ends up needing `serde_json` will use the same version.